### PR TITLE
MFA MUA V1 swagger changes

### PR DESCRIPTION
**Pull request by deymadhumanti**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 

If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

No new changes are there. copied content of 2021-02-01 folder in 2021-02-01-preview folder. Will add MFA MUA related APIs on top of this existing json file

In this PR, we have copied the content from stable\2021-02-01 (latest stable swagger json) to preview\2021-02-01-preview folder. Then, updated the swagger version to 2021-02-01-preview in swagger json file and all example json file in preview\2021-02-01-preview folder. Now stable and preview swagger json has same content. Only swagger version is different for both. For all the features which are in preview stage, we are planning to make changes on top this preview\2021-02-01-preview\bms.json file.

### CODE REVIEW

**Comment by deymadhumanti** [comment]
Hi @jianyexi , Can you please review this PR. No new changes are there. Copied content of 2021-02-01 folder in 2021-02-01-preview folder. We will make the swagger changes for those features which are in preview stage on top of this changes.

**Comment by jianyexi** [comment]
> Hi @jianyexi Jianye Xi FTE , Can you please review this PR. No new changes are there. Copied content of 2021-02-01 folder in 2021-02-01-preview folder. We will make the swagger changes for those features which are in preview stage on top of this changes.

It seems have duplicate schema in the swagger , could you fix it ?


❌ PreCheck/DuplicateSchema | "readme":"recoveryservicesbackup/resource-manager/readme.md","tag":"package-2021-02-preview","details":"Duplicate Schema named NewErrorResponse -- properties.error.$ref: undefined => "#/components/schemas/schemas:1020" ; This error can be temporarily avoided by using the 'modelerfour.lenient-model-deduplication' setting. NOTE: This setting will be removed in a future version of @autorest/modelerfour; schemas should be updated to fix this issue sooner than that."
-- | --

**Comment by chiragg4u** [comment]
Please go through the contribution checklist added as part of the review and mark them completed. This is a requirement before review can be continued.

**Comment by filizt** [comment]
@deymadhumanti could you update the contribution checklists above? We can't start the review before the checklist is completed.

**Comment by deymadhumanti** [comment]
> @deymadhumanti could you update the contribution checklists above? We can't start the review before the checklist is completed.

Hi @filizt , I have completed the contribution checklist. Can you please review the PR now.

**Comment by filizt** [comment]
>     "version": "2021-02-01-preview",

Are you adding a new preview API version?

---
Refers to: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/preview/2021-02-01-preview/bms.json:4 in 83c5337. [](commit_id = 83c533790148b27a2ac50fbbebd30b1594d076d7, deletion_comment = False)

**Review by filizt — APPROVED** [review]

File: specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/preview/2021-02-01-preview/examples/BackupResourceEncryptionConfig_Get.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rishgrp",
+    "vaultName": "rishTestVault",
+    "api-version": "2021-01-01"
```
Comment (filizt) [inline specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/preview/2021-02-01-preview/examples/BackupResourceEncryptionConfig_Get.json:6]: > "api-version": "2021-01-01" [](start = 3, length = 28)

nit - update api-version to the new version

**Pull request merged**